ansible/example_playbook.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2015 TWO SIGMA OPEN SOURCE, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- hosts: satellite_masters
+  gather_facts: false
+  vars_files:
```

Comment:
They just set vars (same as if the vars were inline here).  I'm neutral about it, would you prefer to consolidate into the 1 playbook file?  I guess that makes more sense now that this entire example playbook is clearly intended to be mostly "throwaway" whereas the more reusable stuff has been moved to the Roles.